.github/workflows/codecoverage.yml

```diff
@@ -4,43 +4,56 @@ name: CI
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
   push:
-    branches: [ main ]
+    branches: [ main, bug-fix ]
```

Comment:
I think this should only the `main` branch right?

Reply:
yes it should be in main branch. but for testing purpose need to try to run it on other branch.
gonna change back to main after all problem fixed

Reply:
Oh okay. You should add `TODO` here then so you not forget to remove it